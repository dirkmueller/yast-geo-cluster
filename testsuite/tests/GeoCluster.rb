# encoding: utf-8

module Yast
  class GeoClusterClient < Client
    def main
      # testedfiles: GeoCluster.ycp

      Yast.include self, "testsuite.rb"
      TESTSUITE_INIT([], nil)

      Yast.import "Iplb"

      DUMP("GeoCluster::Modified")
      TEST(lambda { GeoCluster.Modified() }, [], nil)

      nil
    end
  end
end

Yast::GeoClusterClient.new.main
