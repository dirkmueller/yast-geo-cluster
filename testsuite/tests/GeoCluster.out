Dump	GeoCluster::Modified
Return	false
